website/content/docs/agent/config/index.mdx

```diff
@@ -51,9 +51,14 @@ required ports and their default settings.
 
 ## Reloadable Configuration
 
-Reloading configuration does not reload all configuration items. The
-items which are reloaded include:
+Some agent configuration options are reloadable at runtime.
+You can run the [`consul reload` command](/commands/reload) to manually reload supported options from configuration files in the configuration directory.
+To configure the agent to automatically reload configuration files updated on disk,
+set the [`auto_reload_config` configuration option](/consul/docs/agent/config/config-files#auto_reload_config) parameter to `true`.
 
+The following agent configuration options are reloadable at runtime: 
+
+Reloadable agent configuration options:
```

Comment:
```suggestion
```
I think this is redundant.

Reply:
Oops. Yeah ... That got in there by accident!